diff --git a/src/ptree.ts b/src/ptree.ts
index 67154f18363f8f3652e616a2a031a37367ee02e5..e1ab6bad5ab45f870fac7b2300093a1f83ff7cf1 100644
--- a/src/ptree.ts
+++ b/src/ptree.ts
@@ -247,7 +247,7 @@ export class ChildProcess<In extends InMask = InMask> {
 	}
 
 	async bytes(): Promise<Uint8Array> {
-		return new Response(this.stdout).bytes();
+		return (new Response(this.stdout) as any).bytes();
 	}
 
 	// ── Wait ─────────────────────────────────────────────────────────────
